blocks/blockstore/blockstore.go

```diff
@@ -7,14 +7,14 @@ import (
 	"sync"
 	"sync/atomic"
 
+	context "context"
```

Comment:
This can be just `"context"` up there with the other stdlib imports, same for the other occurences. Really no blocking issue though.